src/server.ts

```diff
@@ -109,21 +86,6 @@ app.use((req, res, next) => {
   next();
 });
 
-const serve = (p: string, data: string | Buffer, loginServer?: boolean) => {
-  app.get(p, (req, res) => {
-    if (loginServer && !config.loginEnabled) {
```

Comment:
Calling this out - this code was protecting static assets (css, js, favicon, etc).  In the new code, it's unprotected.  I don't think this is a problem.

Reply:
thanks for calling this out. you need this now that javascript has been externalized form token settings.

we should have a system test of some sort in the future that makes sure all of our private routes are in fact private when express is really running.

Reply:
Huh - are you saying that we need to protect the client side JavaScript?  On having system tests for private routes, yes, that seems like a great idea!

Reply:
no. no need to protect client side js. =) just that we have routes that are securred based on the magic ordering of express binding and we should do better ;)